src/web/src/average.ecpp

```diff
@@ -263,8 +264,8 @@ bool database_ready;
                 if (frame) {
                     if (streq (frame, "BAD_REQUEST")) {
                         zstr_free (&frame);
-                        std::string die = "Data for type = '" + checked_type + "', step = '" + checked_step +
-                        "', source = '" + checked_source + "', element_name = " + " '" + element_name + "'.";
+                        std::string die = TRANSLATE_ME ("Data for type = '") + checked_type + "', step = '" + checked_step +
```

Comment:
This doesn't make much sense to me, as 'type' stands for variable name, but other variable names wouldn't be translated. Also it would be unclear to Translator what the sentence would mean, from: Data for type = '